options.go

```diff
@@ -598,6 +598,25 @@ type Options struct {
 		// gets multiplied with a constant of 1 << 16 to yield 1 << 20 (1MB).
 		ReadSamplingMultiplier int64
 
+		// NumDeletionsThreshold defines the minimum number of point tombstones
+		// that must be present in a single data block for that block to be
+		// considered tombstone-dense for the purposes of triggering a
+		// tombstone density compaction. The default value is 100.
+		NumDeletionsThreshold int
+
+		// DeletionSizeRatioThreshold defines the minimum ratio of the size of
+		// point tombstones to the size of the data block that must be reached
+		// for that block to be considered tombstone-dense for the purposes of
+		// triggering a tombstone density compaction. The default value is 0.5.
+		DeletionSizeRatioThreshold float32
+
+		// MinTombstoneDenseBlocks is the minimum number of tombstone-dense
+		// data blocks that must be present in a single table for it to be
+		// eligible for a tombstone density compaction. The default value is 20.
```

Comment:
I think we should tune this by level, as sstables in higher levels are smaller and have fewer data blocks than sstables in lower levels. Something like L0 sstables containing 2 tombstone-dense data blocks would make the sstable eligible for compaction, and for every level down we'd double that threshold so it'd be 64/128 for L5/L6. That also reflects how we double the target file size for sstables as we go down levels.